instance/config.py

```diff
@@ -1,3 +1,5 @@
+SECRET_KEY='mysecretkey'
```

Comment:
Don't publish the `secret_key`. It should be secret. Leverage `os.getenv()` to get it from the environment

Reply:
So a user who clones the repo should make their own secret key?